go.mod

```diff
@@ -9,16 +9,18 @@ require (
 	go.opentelemetry.io/otel v1.15.1
 	go.opentelemetry.io/otel/exporters/prometheus v0.38.1
 	go.opentelemetry.io/otel/metric v0.38.1
-	golang.org/x/crypto v0.6.0
-	golang.org/x/net v0.8.0
-	google.golang.org/grpc v1.54.0
+	golang.org/x/crypto v0.14.0
+	golang.org/x/net v0.17.0
+	google.golang.org/grpc v1.58.3
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/apimachinery v0.27.1
 	k8s.io/klog/v2 v2.100.1
 	k8s.io/kms v0.27.1
 	monis.app/mlog v0.0.4
 )
 
+require google.golang.org/genproto/googleapis/rpc v0.0.0-20230711160842-782d3b101e98 // indirect
```

Comment:
nit: move this down with the other indirect requires? not sure why go adds to a new section